### PR TITLE
User can create a flash card.

### PR DESCRIPTION
**Pull request by ariozone**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+export default class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      cards: []
+    }
+    this.saveCard = this.saveCard.bind(this)
+  }
+  saveCard(card) {
+    this.state.cards.push(card)
```
Comment (thebearingedge) [inline src/app.js]: Be sure not to mutate `state` or its properties directly. Make a copy and use `setState` instead.

File: src/styles.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+.title {
+  font-family: fantasy;
+  font-size: 2.5rem;
+  font-weight: bold;
+  color: gray;
+  text-align: center;
+}
+
+.card-flip {
```
Comment (thebearingedge) [inline src/styles.css]: Please remove all unused CSS.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    "copy-webpack-plugin": "4.5.2",
+    "webpack": "4.20.1",
+    "webpack-cli": "3.1.1",
+    "webpack-dev-server": "3.1.9"
+  },
+  "dependencies": {
+    "@fortawesome/fontawesome-free": "^5.3.1",
+    "react": "16.5.2",
+    "react-dom": "16.5.2",
+    "react-toggle-switch": "^3.0.4",
```
Comment (thebearingedge) [inline package.json]: Please remove all unused dependencies.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, a couple of small changes. Also, please be sure to include a `.gif` with your Pull Request.

**Comment by ariozone** [comment]
create-flash-card

**Comment by ariozone** [comment]
create-flash-card

**Pull request merged**